lib/invoker/power/setup/distro/ubuntu.rb

```diff
@@ -4,6 +4,9 @@ module Invoker
   module Power
     module Distro
       class Ubuntu < Debian
+        def restart_services
+          system('service dnsmasq restart')
+        end
```

Comment:
I was testing on Ubuntu 14.04 LTS which uses upstart. I had an issue where dnsmasq wasn't being restarted during invoker setup. Didn't realize that Ubuntu have moved to systemd now. Will revert this change.